src/WebJobs.Extensions.NotificationHubs/Config/Converter.cs

```diff
@@ -9,19 +10,63 @@ namespace Microsoft.Azure.WebJobs.Extensions.NotificationHubs
 {
     internal static class Converter
     {
+        public static string NotificationPlatform { get; set; }
+
```

Comment:
Can you just pass this into AddNotificationHubConverters as a parameter? That'd be nicer.